ansible/roles/media-licenses/tasks/main.yml

```
@@ -0,0 +1,16 @@
+- name: media licenses sync CronJob ConfigMap
+  kubernetes.core.k8s:
+    kubeconfig: ~/.kube/config
+    namespace: "{{ NAMESPACE }}"
+    template: vidis-sync-cronjob-configmap.yml.j2
+  tags:
+    - configmap
```

the configmap is only need for the cronjob so it shoud also deppend on the `´WITH_MEDIA_LICENSES` condition
```suggestion
- name: media licenses sync CronJob ConfigMap
  kubernetes.core.k8s:
    kubeconfig: ~/.kube/config
    namespace: "{{ NAMESPACE }}"
    template: vidis-sync-cronjob-configmap.yml.j2
    state: "{{ 'present' if WITH_MEDIA_LICENSES is defined and WITH_MEDIA_LICENSES|bool else 'absent'}}"
  tags:
    - configmap
```